code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+
+import { API_KEY } from '../urls';
+
+const MovieDetails = () => {
+    const [MovieDetails, setMovieDetails] = useState({});
```

Comment:
Capitalised names are reserved for components only, not for state properties. Should be `movieDetails`